### ISSUE TITLE
Vimium stops working on google's search results page after restarting browser

### ISSUE THREAD

**Issue by salat-23** [philc/vimium#4537]
Description:
After fresh install, vimium works fine on all google's pages and browser ones (Allow access to file URLs and Allow access to search page results enabled). After closing the browser and opening it up again - it stops working on aforementioned pages.

1. Install vimium
2. Enable Allow access to file URLs and Allow access to search page results in browser extension settings
3. Restart the browser

Chrome version:

Opera	113.0.5230.32 (Official Build) Stable (64-bit) 
Revision	c51f2916e4a14157c276fd33c8bfb24155aac1ad-refs/branch-heads/6533@{#2000}
OS	Linux
JavaScript	V8 12.7.224.18
User Agent	Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 OPR/113.0.0.0

**Comment by UncleSnail** [comment]
When you go into Chrome after restarting, are those options still checked and the addon is still enabled? It sounds like maybe your user profiles folder is being reset upon closing the browser. Is Vimium still working on most normal websites after a restart? Is there anything you can do to get it working again? If so, what?

**Comment by salat-23** [comment]
No, the settings are the same after restart, I also dont have anything user specific as well. It works as normal on any other website, but for some reason google search page just wont respond.

**Comment by mwoz123** [comment]
I recently have noticed same issue - on Ubuntu 24.04 and Fedora 40. Previously it was fine.

Turning it ("Allow to access search results") off and on (in plugin settings) allows to use it but only until next browser start :(

**Comment by mwoz123** [comment]
update: I've checked in Windows 11 - same issue :(

So most probably it's Vimium global issue, no matter the OS.

**Comment by luisvinicius09** [comment]
Having same issue on Opera One(version: 114.0.5282.222) - Windows 10

Disabling and Enabling the extension or the "Allow on search results" options works until next browser restart.

Looking for a fix
